src/api/helpers/map-CIS-countries/map-CIS-country/index.test.ts

```diff
@@ -1,56 +1,64 @@
+import noOnlineInsuranceSupport from './no-online-insurance-support';
 import { EXTERNAL_API_DEFINITIONS, EXTERNAL_API_MAPPINGS } from '../../../constants';
 import { MappedCisCountry } from '../../../types';
-import { mockCisCountries } from '../../../test-mocks';
+import { mockCisCountry } from '../../../test-mocks';
 
 const { CIS } = EXTERNAL_API_DEFINITIONS;
 
 describe('helpers/map-CIS-countries/map-CIS-country', () => {
-  const { 1: initialMockCountry } = mockCisCountries;
-
   const mockCountryBase = {
-    ...initialMockCountry,
-    marketName: initialMockCountry.marketName,
-    riskCategory: EXTERNAL_API_MAPPINGS.CIS.RISK.STANDARD,
-    isoCode: initialMockCountry.isoCode,
+    ...mockCisCountry,
+    marketName: mockCisCountry.marketName,
+    esraClassification: EXTERNAL_API_MAPPINGS.CIS.ESRA_CLASSIFICATION.STANDARD,
+    isoCode: mockCisCountry.isoCode,
     shortTermCoverAvailabilityDesc: CIS.SHORT_TERM_COVER_AVAILABLE.ILC,
     marketRiskAppetitePublicDesc: CIS.NO_COVER,
   };
 
   it('should return an object', () => {
     const result = mapCisCountry(mockCountryBase);
 
+    const esraClassification = mapEsraClassification(mockCountryBase.ESRAClassificationDesc);
+    const shortTermCover = mapShortTermCoverAvailable(mockCountryBase.shortTermCoverAvailabilityDesc);
+    const nbiIssueAvailable = mapNbiIssueAvailable(mockCountryBase.NBIIssue);
+
     const mapped = {
-      name: mockCountryBase.marketName,
+      countryRating: mockCisCountry.countryRatingDesc,
+      esraClassification,
       isoCode: mockCountryBase.isoCode,
-      riskCategory: mapRiskCategory(mockCountryBase.ESRAClassificationDesc),
-      shortTermCover: mapShortTermCoverAvailable(mockCountryBase.shortTermCoverAvailabilityDesc),
-      nbiIssueAvailable: mapNbiIssueAvailable(mockCountryBase.NBIIssue),
+      name: mockCountryBase.marketName,
+      nbiIssueAvailable,
+      shortTermCover,
       canGetAQuoteOnline: false,
       canGetAQuoteOffline: false,
       canGetAQuoteByEmail: false,
       cannotGetAQuote: false,
-      canApplyForInsuranceOnline: false,
-      noInsuranceSupport: false,
     } as MappedCisCountry;
 
-    mapped.canGetAQuoteOnline = canGetAQuoteOnline(mapped);
+    mapped.canGetAQuoteOnline = canGetAQuoteOnline({ shortTermCover, nbiIssueAvailable, esraClassification });
+
     mapped.canGetAQuoteOffline = canApplyOffline(mockCountryBase.shortTermCoverAvailabilityDesc);
-    mapped.canGetAQuoteByEmail = canGetAQuoteByEmail(mapped);
-    mapped.cannotGetAQuote = cannotGetAQuote(mapped);
 
-    mapped.canApplyForInsuranceOnline = applyForInsuranceOnline(mapped.shortTermCover);
+    mapped.canGetAQuoteByEmail = canGetAQuoteByEmail({ shortTermCover, nbiIssueAvailable, esraClassification });
+
+    mapped.cannotGetAQuote = cannotGetAQuote({ shortTermCover, nbiIssueAvailable, esraClassification });
+
+    mapped.canApplyForInsuranceOnline = canApplyForInsuranceOnline(mockCisCountry);
```

Comment:
not an issue, but could these not go in the mapped object on line 34?

Reply:
Updated.